### PR TITLE
fix #343 : create documentation to deploy fluentd

### LINKED ISSUE #343: Document fluentd deployment
I just deploy [fluentd](https://www.fluentd.org/) on clever cloud.
I take me some time to understand how to deploy it regarding clever restrictions on ruby and docker runtime, so I suggest to create a doc for this.

This issue aims to track pull request and decide what is needed in the doc

Proposition : 
- [ ] Add a quick presentation of the solution
- [ ] Create the doc in the docker runtime section, since ruby runtimes don't allow to deploy it
- [ ] Describe installation from gem schema in a Docker runtime (in fact it's quite easy when you have the solution :) 
- [ ] Describe a minimal conf = worker run on clever without plugins or actual collector

### PR DESCRIPTION
**Pull request by oteku**
Committer: Thomas Haessle <thomas.haessle@gmail.com>

### CODE REVIEW

File: runtimes/docker/fluentd.md [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+shortdesc: How to deploy a fluentd using Docker on Clever Cloud.
+tags:
+- docker
+- ruby
+- fluentd
+---
+
+## Introduction
+
+Since you deploy microservice on on Clever Cloud you may need some data pipes between your services to :
```
Comment (BlackYoup) [inline runtimes/docker/fluentd.md]: double `on`

File: runtimes/docker/fluentd.md [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+
+* collect datas from your PostrgreSQL to create Elasticsearch indexes for your website search engine
```
Comment (BlackYoup) [inline runtimes/docker/fluentd.md]: `data` is already a plural, no need for an `s` here

File: runtimes/docker/fluentd.md [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+* collect application logs to analyze them with elasticsearch, then archive them with S3
+* collect Apache access logs to analyze them in MongoDB
+* extract datas from the database of your PHP/MySQL application to transform then load them in your other node.js/PostgreSQL application
```
Comment (BlackYoup) [inline runtimes/docker/fluentd.md]: Same here for `datas`

File: runtimes/docker/fluentd.md [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+
+* Docker
+
+* Git
+
+* curl
+
+<div class="alert">
+To manage your gems and ruby versions, we recommend <a href="https://github.com/sstephenson/rbenv">rbenv</a>.
+
```
Comment (BlackYoup) [inline runtimes/docker/fluentd.md]: The alert div is never closed

File: runtimes/docker/fluentd.md [hunk 5]
```diff
@@ -0,0 +1,127 @@
...
+## Create a fluentd application on your local
```
Comment (BlackYoup) [inline runtimes/docker/fluentd.md]: s/on your local/locally/ ?

File: runtimes/docker/fluentd.md [hunk 6]
```diff
@@ -0,0 +1,127 @@
...
+gem 'fluent-plugin-td'
+```
+
+Then run bundler to install dependancies and generate your `Gemfile.lock`
+
+```bash
+$ bundle install
+```
+
+Clever Cloud need that your application answer on request made on **0.0.0.0:8080**, we'll use a PORT environment variable for local test purpose (this variable is automatically setup on each application).
```
Comment (BlackYoup) [inline runtimes/docker/fluentd.md]: needs*, answers*, requests*

Maybe put the PORT between ` ?

**Comment by oteku** [comment]
Désolé pour la revue tardive, j'ai commité les propositions de fix de @urcadox

**Comment by urcadox** [comment]
LGTM, ping @cnivolle

**Pull request merged**